res/stylesheet.qss

```diff
@@ -75,10 +75,16 @@ QPushButton#SUV2ROI_Button {
 }
```

Comment:
I know most of your dev is on MAC, do we need someone to test it on windows to double-check this stylesheet?

Reply:
 Don't worry about this, I wrote everything on a windows desktop(I major use windows), and tested everything. Fix the mac stylesheet issue just a  small bug I found when I tested on my MacBook. Just need to get someone to test it on Linux or I can test it after I finish work so should be fine.

Reply:
Im on Ubuntu :), worked for me so all good then!